src/Network/Minio/S3API.hs

```diff
@@ -550,8 +551,8 @@ headObject bucket object reqHeaders = do
           riBucket = Just bucket,
           riObject = Just object,
           riHeaders = reqHeaders
+            <> [("Accept-Encoding", "identity")]
```

Comment:
Maybe add a comment here for future readers as reference would help, so that it's not removed by mistake.

Reply:
Fixed!